Dart/src/com/jetbrains/lang/dart/ide/runner/test/DartTestRunningState.java

```diff
@@ -32,6 +37,8 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import static com.intellij.openapi.command.impl.CommandLog.LOG;
```

Comment:
I guess this import was suggested by too smart IntelliJ's code completion :). In fact loggers from other classes rarely should be reused. I'll change it to current class own logger.